examples/usdConverter/README.md

```diff
@@ -6,33 +6,34 @@ This example shows how a world in a SDF file can be converted to [USD](https://g
 
 You will need all of the dependencies for sdformat, along with the following additional dependencies:
```

Comment:
We should mention that SDFormat itself is also needed, now that cloning it isn't part of the tutorial anymore

Reply:
https://github.com/gazebosim/sdformat/pull/1094/commits/f3daf6e9c744d7d1fde87ccf13ca76695f79e008